packages/react-core/src/demos/Animations/examples/Animations.tsx

```diff
@@ -0,0 +1,1699 @@
+          Dashboard
+        </NavItem>
+        <NavExpandable title="Resources" groupId="resources-group" isActive={activeGroup === 'resources-group'}>
+          <NavItem
+            preventDefault
+            id="overview"
+            to="#overview"
+            groupId="resources-group"
+            itemId="overview"
+            isActive={activeItem === 'overview'}
+          >
+            Overview
+          </NavItem>
+          <NavItem
+            preventDefault
+            id="health"
+            to="#health"
+            groupId="resources-group"
+            itemId="health"
+            isActive={activeItem === 'health'}
+          >
+            Health
+          </NavItem>
+        </NavExpandable>
+      </NavList>
+    </Nav>
+  );
+  const kebabDropdownItems = (
+    <>
+      <DropdownItem>
+        <CogIcon /> Settings
+      </DropdownItem>
+      <DropdownItem>
+        <HelpIcon /> Help
+      </DropdownItem>
+    </>
+  );
+  const userDropdownItems = (
+    <>
+      <DropdownItem key="group 2 profile">My profile</DropdownItem>
+      <DropdownItem key="group 2 user">User management</DropdownItem>
+      <DropdownItem key="group 2 logout">Logout</DropdownItem>
+    </>
+  );
+  const headerToolbar = (
+    <Toolbar>
+      <ToolbarContent>
+        <ToolbarGroup align={{ default: 'alignEnd' }}>
+          <ToolbarGroup variant="action-group-plain">
+            <ToolbarItem visibility={{ default: 'visible' }} selected={isDrawerExpanded}>
+              <NotificationBadge
+                variant={getNumberUnread() === 0 ? 'read' : 'unread'}
+                onClick={(event: any) => onCloseNotificationDrawer(event)}
+                aria-label="Notifications"
+                isExpanded={isDrawerExpanded}
+                count={getNumberUnread()}
+                shouldNotify={shouldNotify}
+              />
+            </ToolbarItem>
+            <ToolbarGroup
+              variant="action-group-plain"
+              visibility={{
+                default: 'hidden',
+                lg: 'visible'
+              }} /** the settings and help icon buttons are only visible on desktop sizes and replaced by a kebab dropdown for other sizes */
+            >
+              <ToolbarItem>
+                <Button
+                  aria-label="Settings actions"
+                  className="pf-m-settings"
+                  variant={ButtonVariant.plain}
+                  icon={<CogIcon />}
+                />
+              </ToolbarItem>
+              <ToolbarItem>
+                <Button aria-label="Help actions" variant={ButtonVariant.plain} icon={<QuestionCircleIcon />} />
+              </ToolbarItem>
+            </ToolbarGroup>
+          </ToolbarGroup>
+          <ToolbarGroup>
+            <ToolbarItem
+              visibility={{
+                default: 'hidden'
+              }} /** this kebab dropdown replaces the icon buttons and is hidden for desktop sizes */
```

Comment:
With being set to hidden by default and no other visibility options won't this just never render?